Library/Formula/terminal-notifier.rb

```diff
@@ -27,6 +27,6 @@ def install
   end
 
   test do
-    system "#{bin}/terminal-notifier", "-message", "Hello World!"
+    system "#{bin}/terminal-notifier", "-help"
```

Comment:
> I don't think if MacOS.version < :yosemite is a good fix. The test failed is not Yosemite specific. It is related to our CI settings. So in the future, this test can fail again in different OS.

It's not related to our CI settings, it's related to starting graphical applications over SSH due to changes in Yosemite so I think this change is correct.



> It would be great that if you can investigate why this fails in the headless environment(like invoking command over ssh). Or can you just stop terminal-notifier connecting to WindowServer when only asking help message.

Again, I don't really agree here. It's not specific to terminal-notifier so I don't think there's any changes you need to make @julienXX.



> If we cannot fix it, I would suggest to kill test completely. From the git log, we did this before in this formula.

I'd rather a test on some platforms than none.